future-ninjas/src/Components/JobsPage/JobsDiplay/index.js

```diff
@@ -0,0 +1,252 @@
+  margin-right: 1em; 
+  display: flex;
+  justify-content: initial;
+  align-items: center;
+  flex-direction: column;
+  @media (min-width: 1600px) {
+    height: 87.5vh;
+  }
+  @media (min-width: 1920px) {
+    height: 88vh;
+  }
+  @media (min-width: 2560px) {
+    height: 88.9vh;
+  }
+`
+
+const JobsArea = styled.div`
+  background-color: #F5F3FB;
+  width: 80%;
+  border-radius: 15px;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
+  grid-gap: 1em;
+  justify-items: center;
+  align-items: center;
+  padding: 1em 0.5em;
+  @media (min-width: 1600px) {
+    height: 83.9vh;
+  }
+  @media (min-width: 1920px) {
+    height: 85vh;
+  }
+  @media (min-width: 2560px) {
+    height: 86vh;
+    font-size: 18pt;
+  }
+`
+
+const Title = styled.h4`
+  text-align: center;
+  @media (min-width: 1920px) {
+    font-size: 15pt;
+  }
+  @media (min-width: 2560px) {
+    font-size: 18pt;
+  }
+`
+
+const StyledLabel = styled.label`
+  margin-bottom: 0.3em;
+`
+
+const StyledInput = styled.input`
+  border: 1px solid #D4D4D4;
+  box-sizing: border-box;
+  border-radius: 6px;
+  height: 2.5em;
+  width: 100%;
+  text-indent: 0.5em;
+  margin-bottom: 1em;
+  @media (min-width: 1920px) {
+    font-size: 15pt;
+  }
+  @media (min-width: 2560px) {
+    font-size: 18pt;
+  }
+`
+
+const StyledSelect = styled.select`
+  border: 1px solid #D4D4D4;
+  box-sizing: border-box;
+  border-radius: 6px;
+  height: 2.5em;
+  width: 100%;
+  text-indent: 0.5em;
+  margin-bottom: 1em;
+  @media (min-width: 1920px) {
+    font-size: 15pt;
+  }
+  @media (min-width: 2560px) {
+    font-size: 18pt;
+  }
+`
+
+const ContainerSearch = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 88%;
+  @media (min-width: 1920px) {
+    font-size: 15pt;
+  }
+  @media (min-width: 2560px) {
+    font-size: 18pt;
+  }
+`
+
+
+
+export default class JobsDisplay extends React.Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      minValue: null,
+      maxValue: Infinity,
+      title: '',
+      description: '',
+      allJobs: [],
+      order: '',
+    }
+  }
+
+  changeMinValue = (el) => {
+    this.setState({ minValue: el.target.value })
+  }
+
+  changeMaxValue = el => {
+    if (el.target.value === '') {
+      this.setState({ maxValue: Infinity })
+    } else {
+      this.setState({ maxValue: el.target.value })
+    }
+  }
+
+  changeTitle = el => {
+    this.setState({ title: el.target.value });
+  }
+
+  changeDescription = el => {
+    this.setState({ description: el.target.value });
+  }
+
+  componentDidMount() {
+    this.getAllJobs()
+  }
+
+  getAllJobs = async () => {
+    axios.get(`https://us-central1-future-apis.cloudfunctions.net/futureNinjas/jobs`).then(res => {
+      const allJobs = res.data.jobs
+      this.setState({ allJobs })
+    })
+  }
+
+
+  filtroServico = () => {
+    return this.state.allJobs.filter((preco) => {
+      if (preco.value >= Number(this.state.minValue)) {
```

Comment:
Tente usar a abreviação de: 
`return preco.value >= Number(this.state.minValue)`